Misc/NEWS.d/next/Library/2025-10-20-09-12-18.gh-issue-140344.WjbYg-.rst

```diff
@@ -0,0 +1,3 @@
+The classes ``ast.Suite``, ``ast.AugLoad``, ``ast.AugStore``, and
```

Comment:
This only lists four of the seven things deprecated. It also doesn't mention the `.dims` attribute.